docs/rst/notes/new_version.rst

```diff
@@ -6,36 +6,6 @@
 Forthcoming Version
 ###################
 
-Next release will include the following **Configuration features**:
+*"To improve is to change; to be perfect is to change often."*
 
-* Support TLS configuration and communication.
-* Support IPv6 communication via UDP, TCP and TLS.
-* Support DNS by given Domain Name in configuration instead of an IP address.
-* Support keyed topics.
-
-This release includes the following **Routing features**:
-
-* Zero-Copy data transmission between internal Participants.
-
-Next release will include the following **User Interface features**:
-
-* Allow to close the Router application gently with ``SIGTERM`` (``kill``) as with ``SIGINT`` (``^C``).
-
-Next release will include the following **Continuous-Integration features**:
-
-* Add communication tests for UDP, TCP and TLS WAN cases.
-* Extend tool test with more complex configurations.
-* Erase Flaky tests from CI required passing tests.
-* Implement a new class to check that no warning or error logs are produced during test executions.
-* Add gMock to test libraries.
-
-Next release will fix the following **major bugs**:
-
-* Fix GUID creation when explicit guid is provided.
-
-Next release will fix the following **minor bugs**:
-
-* Change ``YAML`` example configurations to ``YAML`` format (instead of ``JSON``) fixing
-  an issue when blank lines were missing at the end of the file.
-* Normalize the error and process exit when failure.
-* Fix documentation typos.
+Winston S. Churchill
```

Comment:
I'd remove this section for the release so that it won't appear in `v0.2.0` of the documentation. Then we can add it again to be visible just in the `latest` version.

Reply:
then I would remove it in v0.2.0 branch (the one that will change ddsrouter.repos)

Reply:
I recommend you to leave that branch as it is. Then, when everything is ready for the release we rebase it to `main` with just the changes in the `.repos` file. That way we don't miss anything.